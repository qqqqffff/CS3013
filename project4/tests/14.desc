eval SJF analysis with one job
