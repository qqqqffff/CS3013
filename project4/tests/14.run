./scheduler SJF tests/14.in 0
