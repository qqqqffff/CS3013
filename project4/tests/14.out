Begin analyzing SJF:
Job 0 -- Response time: 0  Turnaround: 4  Wait: 0
Average -- Response: 0.00  Turnaround: 4.00  Wait: 0.00
End analyzing SJF.
